tests/misc_env/install_prereq.py

```diff
@@ -173,6 +184,41 @@ def enable_rhel_rpms(ceph, distro_ver):
         )
 
 
+def enable_rhel_eus_rpms(ceph, distro_ver):
+    """
+    Setup cdn repositories for rhel systems
+    Args:
+        distro_ver: distro version details
+        ceph: ceph object
+    """
+
+    eus_repo = {"7": ("7.7", ["rhel-7-server-eus-rpms"])}
+
+    ceph.exec_command(
+        sudo=True,
+        cmd="subscription-manager release --set={distro}".format(
+            distro=eus_repo.get(distro_ver[0])[0]
+        ),
+        long_running=True,
+    )
+
+    for repo in eus_repo.get(distro_ver[0])[1]:
+        ceph.exec_command(
+            sudo=True,
+            cmd="subscription-manager repos --enable={r}".format(r=repo),
```

Comment:
Note that users must disable `rhel-7-server-rpms` and then enable `rhel-7-server-eus-rpms`. It's not clear to me if this code might lead to a situation where a server has both repos enabled.

Reply:
we disable all the rpms before enabling any rpms [here](https://github.com/red-hat-storage/cephci/blob/master/tests/misc_env/install_prereq.py#L76). that's why I skipped the disabling part